packages/react-components/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -395,19 +404,19 @@ const useRootFocusStyles = makeStyles({
 const useRootIconOnlyStyles = makeStyles({
   // Size variations
   small: {
-    ...shorthands.padding(tokens.spacingHorizontalXS),
+    ...shorthands.padding(buttonSpacingSmallWithIcon),
 
-    minWidth: '28px',
-    maxWidth: '28px',
+    minWidth: '24px',
+    maxWidth: '24px',
```

Comment:
nit: Are these minWidth/maxWidth still necessary? The button should be the right size based on the icon size plus padding and border, right?

Reply:
I'll check if I can get these without specifying minWidth/maxWidth in a separate PR to not delay the merging of this one.